scripts/helperscripts/inc.networkHelper.sh

```diff
@@ -0,0 +1,123 @@
+_get_service_enablement() {
```

Comment:
Sure. 
The only other place i found that is using this logic, is in the installation check script. But thats slightly different.
Do you have any other place in mind that would be useful?

Otherwise leave it there for now and refactor if we have a new usecase?